bin/postgres_common/start.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Start script for compacted postgres operator image
+# Used to run correct start script based on MODE
+
+CRUNCHY_DIR=${CRUNCHY_DIR:-'/opt/crunchy'}
+source $CRUNCHY_DIR/bin/common_lib.sh
+enable_debugging
+
+env_check_err "MODE"
+
+echo_info "Image mode found: ${MODE}"
+case $MODE in 
+    postgres)
+      echo_info "Starting in 'postgres' mode"
+      /bin/bash $CRUNCHY_DIR/bin/postgres/start.sh
+      ;;
+    pgdump)
+      echo_info "Starting in 'pgdump' mode"
+      /bin/bash $CRUNCHY_DIR/bin/pgdump/start.sh
+      ;;
+    pgrestore)
+      echo_info "Starting in 'pgrestore' mode"
+      /bin/bash $CRUNCHY_DIR/bin/pgrestore/start.sh
+      ;;
+    pgbench)
+      echo_info "Starting in 'pgbench' mode"
+      /bin/bash $CRUNCHY_DIR/bin/pgbench/start.sh
+      ;;
+    pgbasebackup-restore)
+      echo_info "Starting in 'basebackup-restore' mode"
+      /bin/bash $CRUNCHY_DIR/bin/pgbasebackup_restore/start.sh
+      ;;
+    backup)
```

Comment:
since the backup mode uses `pg_basebackup` maybe we use this as a time to make the `pgbasebackup-restore` and `backup` modes match. (`pgbasebackup-backup`)